src/api/app/views/webui2/webui/project/_side_links.html.haml

```diff
@@ -0,0 +1,50 @@
+%ul.list-unstyled
+  - if is_maintenance_project
+    = render partial: 'webui2/webui/project/side_links/maintenance_project', locals: { open_maintenance_incidents: open_maintenance_incidents,
+                                                                                       project: project,
+                                                                                       maintained_projects: maintained_projects }
+  - else
```

Comment:
`elsif` :see_no_evil:

Reply:
No because there are multiple `if` under that `else`

Reply:
I need new glasses... :nerd_face: